apps/journeys-admin/src/components/Editor/Slider/Content/Goals/GoalsList/GoalsList.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import { Meta, StoryObj } from '@storybook/react'
+import { ComponentProps } from 'react'
+
+import { GoalType } from '@core/journeys/ui/Button/utils/getLinkActionGoal/getLinkActionGoal'
+
+import { journeysAdminConfig } from '../../../../../../libs/storybook'
+
+import { GoalsList } from './GoalsList'
+
+
+const GoalsListStory: Meta<typeof GoalsList> = {
+  ...journeysAdminConfig,
+  component: GoalsList,
+  title: 'Journeys-Admin/Editor/Slider/Content/Goals/GoalsList'
+}
+
+type Story = StoryObj<ComponentProps<typeof GoalsList> >
+
+const Template: Story = {
+  render: (args) => (
+    <GoalsList goals={args.goals} />
```

Comment:
Since you have already typed args (from line 17) you can just pass it all into the component
```suggestion
    <GoalsList {...args} />
```